### PR TITLE
Feature/grape

### PR DESCRIPTION
**Pull request by sarahverbelen**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Grapes</h1>
```
Comment (guevarajoge) [inline index.html:9]: change to other fruit

**Comment by eliseboes** [comment]
Accepted.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
```
Comment (wienertgillijns) [inline index.html:8]: Mooi zo!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    <h1>Grapes</h1>
+    <p>A grape is a fruit, botanically a berry, of the deciduous woody vines of the flowering plant genus Vitis. Grapes can be eaten fresh as table grapes or they can be used for making wine, jam, grape juice, jelly, grape seed extract, raisins, vinegar, and grape seed oil.</p>
```
Comment (eliseboes) [inline index.html:10]: Shorten the length of the text.

**Review by eliseboes — COMMENTED** [review]
Shorten the length of the text.

File: index.html (hunk 2 above)
Comment (wienertgillijns) [reply index.html:8]: Chique

**Pull request closed without merging**